tests/test_harvester_daily_bfg_prateb.py

```diff
@@ -72,118 +75,57 @@ def test_global_mean_values_offline(tolerance=0.001):
         bfg_1994010200_fhr06_prateb_control.nc
         
     When averaged together, these files represent a 24 hour mean. The 
-    average value hard-coded in this test was calculated from 
+    average values hard-coded in this test were calculated from 
     these forecast files using a separate python code.
     """
     data1 = harvest(VALID_CONFIG_DICT)
-    global_mean = 3.1173840683271906e-05
-    assert data1[0].value <= (1 + tolerance) * global_mean
-    assert data1[0].value >= (1 - tolerance) * global_mean
-
-def test_global_mean_values_netCDF4(tolerance=0.001):
-    """Opens each background Netcdf file using the
-    netCDF4 library function Dataset and computes the expected value
-    of the provided variable.  In this case prateb_ave.
-    """
-    data1 = harvest(VALID_CONFIG_DICT)
-    
-    gridcell_area_data = Dataset(GRIDCELL_AREA_DATA_PATH)
-    norm_weights = gridcell_area_data.variables['area'][:] / np.sum(
-                                        gridcell_area_data.variables['area'][:])
-    
-    summation = np.ma.zeros(gridcell_area_data.variables['area'].shape)
-    for file_count, data_file in enumerate(BFG_PATH):
-        test_rootgrp = Dataset(data_file)
-    
-        summation += test_rootgrp.variables[VALID_CONFIG_DICT['variable'][0]][0]
-        
-        test_rootgrp.close()
-        
-    temporal_mean = summation / (file_count + 1)
-    global_mean = np.ma.sum(norm_weights * temporal_mean)    
-    
+    if 'region' in str(data1[0]):
+        global_means = [7.04825819962097e-05, 2.8965191456702544e-05,2.2748022739982528e-05, \
+                        2.2748022739982528e-05,3.1183886948414354e-05]
+    iregion = 0 
     for i, harvested_tuple in enumerate(data1):
         if harvested_tuple.statistic == 'mean':
-            assert global_mean <= (1 + tolerance) * harvested_tuple.value
-            assert global_mean >= (1 - tolerance) * harvested_tuple.value
-            
-    gridcell_area_data.close()
-                
+           assert global_means[iregion] <= (1 + tolerance) * harvested_tuple.value
+           assert global_means[iregion] <= (1 + tolerance) * harvested_tuple.value
```

Comment:
this assert statement is the exact same as the one above it. please remove the duplicate or change it to check something else